lca-genesys-audiohook-stack/README.md

```diff
@@ -77,3 +85,10 @@ Now, follow the below steps to enable the Audio hook into Gensys Cloud flows.
 5. Click on the Call ID to see the call details page that includes live turn-by-turn transcripts and sentiment trends
 
 ***
+
+## Genesys Custom Application Integration - Live Call Analytics 
+To streamline access for LCA application instead of swith between Genesys Cloud and LCA, you can now setup a custom applicaiton integration in Genesys.
```

Comment:
Typo in 'swith'.  No worries, I will fix.

Reply:
Thanks so much @anilchinnam !!  I will merge to develop and it'll be in the next release.